draft-ietf-quic-recovery.md

```diff
@@ -463,9 +466,9 @@ Pseudocode for OnLossDetectionAlarm follows:
 ## Detecting Lost Packets
 
 Packets in QUIC are only considered lost once a larger packet number is
-acknowledged.  DetectLostPackets is called every time there is a new largest
-packet or if the loss detection alarm fires the previous largest acked packet is
-supplied.
+acknowledged.  DetectLostPackets is called every time an ack is received.
+If the loss detection alarm fires and the lost_time is set, the previous
```

Comment:
Make loss_time or lost_time consistent. Either works. How about "reordering_threshold_in_time" and "reordering_threshold_in_packets" (or remove the "in" if you want more succinct) instead of loss_time and reordering_threshold ?